index.html

```diff
@@ -780,9 +780,43 @@ <h2 id="4Concepts.Sourceimage">Images</h2>
         <!-- Maintain a fragment named "3channel" to preserve incoming links to it -->
         <dfn id="3channel">channel</dfn>. Each channel has a sample depth in the range 1 to 16, which is the number of bits used by every sample in the
         channel. Different channels may have different sample depths. The red, green, and blue samples determine the intensities of
-        the red, green, and blue components of the pixel's color; if they are all zero, the pixel is black, and if they all have
+        the red, green, and blue components of the pixel's color. 
+        
+        <p>The following table describes the quantization of image black to image peak white (signal range). By default the signal range is full-range 
+          but it can also be represented as narrrow range, which conforms to quantization in broadcast video which typically uses the YCbCr Color 
+          Representation.  cICP can specify this specific range:</p>
+
+          <br>
+        <table id="signal-range-syntax" class="numbered simple">
+          <caption>
+            Full and Narrow Range Signal Range (Quantization Method)
+          </caption>
+
+         
+          <tr>
+            <th>16bit Signal Range (Quantization for Full and Narrow Ranges)</th>
+            <th>Black</th>
+            <th>White</th>
+          </tr>
+
+          <table border="1">
```

Comment:
I think this line can be removed. It accidentally adds an extra table inside the other table. Notice, it also never gets closed.